src/benchmark/frameworks/openvino/openvino_benchmark_process.py

```diff
@@ -101,6 +102,8 @@ def __init__(self, test, executor, log, cpp_benchmarks_dir, perf_hint=''):
         if not self._benchmark_path.is_file():
             raise invalid_path_exception
 
+        self._report_path = executor.get_path_to_logs_folder().joinpath('benchmark_report.json')
```

Comment:
why not the same log name with timestamp as in ORT?

Reply:
afaik openvino benchmark_app receives only report_folder, report name is always benchmark_report.json if json format is enabled

Reply:
Oh, ok then